src/clean.js

```diff
@@ -51,6 +51,22 @@ function clean(node, newObj) {
   if (["Location", "Position"].includes(node.constructor.name)) {
     newObj = "locationNode";
   }
+
+  if (["foreach", "for", "if", "while", "do"].indexOf(node.kind) !== -1) {
```

Comment:
instead of `.indexOf` we could use `.includes`, like in the previous `if` statement?

Reply:
I'm not sure about nodejs internal optimisations, but in principle the fact to build an array at each time the if statement is executed should cost more than defining a const array :

```js
// in header, outside function
const statements = ["foreach", "for", "if", "while", "do"];
// into a function/loop repeated many times 
if (statements.includes(node.kind) {
```

Reply:
it would not surprise me if v8 optimised this away (it should be able to see that the array does not change), but I have no idea either.

Reply:
let's do it thanks :+1: